addons/html_builder/static/src/core/field_change_replication_plugin.js

```diff
@@ -0,0 +1,128 @@
+import { Plugin } from "@html_editor/plugin";
+import { closestElement, selectElements } from "@html_editor/utils/dom_traversal";
+import { withSequence } from "@html_editor/utils/resource";
+
+export class FieldChangeReplicationPlugin extends Plugin {
+    static id = "fieldChangeReplication";
+
+    resources = {
+        handleNewRecords: this.handleMutations.bind(this),
+        normalize_handlers: withSequence(9000, this.normalizeHandler.bind(this)),
+    };
+
+    setup() {
+        this.fieldsToReplicate = new Set();
+        this.ignoredClasses = new Set(this.getResource("system_classes"));
+        this.ignoredAttrs = new Set(this.getResource("system_attributes"));
+        this.ignoreClassesSelector = [...this.ignoredClasses].map((c) => "." + c).join(",");
+        this.ignoredAttrsSelector = [...this.ignoredAttrs].map((a) => "[" + a + "]").join(",");
+    }
+
+    /**
+     * @typedef { import("./history_plugin").HistoryMutationRecord } HistoryMutationRecord
+     *
+     * @param { HistoryMutationRecord[] } records
+     */
+    handleMutations(records) {
+        records
+            .filter((r) => !(r.type === "attributes" && r.attributeName.startsWith("data-oe-t")))
+            .map((r) =>
+                closestElement(r.target, "[data-oe-model], [data-oe-translation-source-sha]")
+            )
+            .filter(Boolean)
+            // Do not forward "unstyled" copies to other nodes.
+            .filter((field) => !field.classList.contains("o_translation_without_style"))
```

Comment:
fieldEl

Reply:
you only have this class "o_translation_without_style" in translate mode or in edit too ?